.gitignore

```diff
@@ -1,2 +1,3 @@
 /vendor
 composer.lock
+.idea
```

Comment:
this should be part of your own local gitignore, since it has nothing to do with the project, but with your local development environment